kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/assertions/nondeterministic/eventually.kt

```diff
@@ -115,7 +115,12 @@ data class EventuallyConfiguration(
    val listener: EventuallyListener,
    val shortCircuit: (Throwable) -> Boolean,
    val includeFirst: Boolean,
-)
+) {
+   init {
+      require(duration.inWholeMilliseconds >= 0) { "Duration must be greater than or equal to 0" }
+      require(retries >= 0) { "Retries must be greater than or equal to 0" }
+   }
```

Comment:
```suggestion
   init {
      require(duration.inWholeMilliseconds >= 0) { "Duration must be greater than or equal to 0, but was $duration" }
      require(retries >= 0) { "Retries must be greater than or equal to 0, but was retries" }
   }
```

Suggestion: Add the actual values to the messages, so the error messages are easier to understand.

Reply:
should be `but was $retries" }` not `but was retries" }`

Reply:
oops, thanks @AlexCue987 - fixed